plugins/woocommerce-admin/client/marketing/overview-multichannel/IntroductionBanner/IntroductionBanner.tsx

```diff
@@ -42,13 +46,19 @@ export const IntroductionBanner = ( {
 	 * and it does not make sense to display the "Add channels" button in this introduction banner
 	 * that will do nothing upon click.
 	 *
+	 * This also depends on the `showCreateCampaignButton` above.
+	 * If "create campaign" button is not shown, then "add channels" button should not be shown either.
+	 * This is to make it look better in the UI.
```

Comment:
```suggestion
	 * This depends on the `showCreateCampaignButton`.
	 * If the "create campaign" button is not shown, then the "add channels" button should not be shown either.
```

Or remove entirely if you will use the other suggestion I have for this.

Reply:
Thanks, I have removed it in [`04294c4` (#38825)](https://github.com/woocommerce/woocommerce/pull/38825/commits/04294c4ec8301ab424bff723b54a8f770cf2863a).